PopupBridge/Classes/POPPopupBridge.m

```diff
@@ -55,7 +55,7 @@ - (id)initWithWebView:(WKWebView *)webView delegate:(id<POPPopupBridgeDelegate>)
 
                 NSMutableDictionary *payloadDictionary = [NSMutableDictionary new];
                 payloadDictionary[@"path"] = path;
-                payloadDictionary[@"queryItems"] = [self.class dictionaryForQueryString:url.query];
+                payloadDictionary[@"queryItems"] = [POPPopupBridge dictionaryForQueryString:url.query];
```

Comment:
These instances of `self.class` should be changed to `weakSelf.class`. In general we want to avoid using hard coded class names because it can break sub-classing by preventing method overriding.

Reply:
We figured it was OK to use the class name in this case since these are private methods that can't be overridden in a subclass. But I'm happy to change it to `weakSelf.class` if you prefer.

Reply:
Updated ✔️